apps/site/assets/js/algolia-result.js

```diff
@@ -220,6 +227,7 @@ function getPopularIcon(icon) {
 }
 
 export function getIcon(hit, type) {
+  console.log("in getIcon " + hit._highlightResult.route.name.value);
```

Comment:
`hit.route` returns the Silver Line route here for me (at least which the searches I've been trying) --  and yep` _iconFromRoute`(in line 238) is currently returning "bus" for it -- so we'll have to add an extra bit of logic so it returns something different. I'd try inspecting/adjusting `_iconFromRoute` function so that it returns `silver_line`

Reply:
What stops are you searching? I've been trying to use `south` and `Chelsea`

Reply:
What searches were you using? I've tried `Chelsea` and `South`

Reply:
`Silver` (to pull up the Silver line route in the dropdown)